tests/test_quantum.py

```diff
@@ -54,6 +56,46 @@ def testConstructor(self):
         self.assertEqual(quantum.endTime, endTime)
         self.assertEqual(quantum.host, host)
 
+    def testAddInputsOutputs(self):
+        """Test of addPredictedInput() method.
+        """
+        quantum = Quantum(task="some.task.object", run=None)
+
+        # start with empty
+        self.assertEqual(quantum.predictedInputs, dict())
+
+        camera = "DummyCam"
+        dsName = "test_ds"
```

Comment:
`datasetTypeName`. I know it is verbose, but I like verbose :) And it is used all over the place.